packages/theme/src/components/donut-chart.ts

```diff
@@ -0,0 +1,36 @@
+import { mergeMultiStyle, type ComponentMultiStyle } from "@yamada-ui/core"
+import { LineChart } from "./line-chart"
+
+export const DonutChart: ComponentMultiStyle = mergeMultiStyle(LineChart, {
+  baseStyle: {
+    donut: {},
+    activeShape: {},
+    inactiveShape: {},
+    label: { fillOpacity: 1 },
+    labelLine: {},
+    cell: {},
+  },
+
+  sizes: {
+    sm: {
+      container: {
+        w: "full",
+      },
+    },
+    md: {
+      container: {
+        w: "full",
+      },
+    },
+    lg: {
+      container: {
+        w: "full",
+      },
+    },
+    full: {
```

Comment:
It seems that the content is duplicated with `LineChart`. This should either be deleted or be an empty object.